### ISSUE TITLE
user based permission 

### ISSUE THREAD

**Issue by rylxes** [Zizaco/entrust#889]
Can user based permission be implemented with entrust ?, ... like assigning permissions to users instead of roles

**Comment by Otisz** [comment]
When a user registers, attach a default role to the user. That would be my solution.

**Comment by faiverson** [comment]
No, you cannot. The relation is a user has role/s. A role has permission/s.
A default role like the suggestion above or a role per user might work.
